messages/lightning.preview.site.md

```diff
@@ -1,38 +1,37 @@
 # summary
 
-Preview Experience Cloud Sites.
+Preview an Experience Cloud site locally and in real-time, without deploying it.
 
 # description
 
-In dev preview mode, you can edit local files and see these changes to your Lightning Web Components (LWC) within your site:
+In developer preview mode, see local changes to your site in a real-time preview that you don't have to deploy or manually refresh. To let you quickly iterate on your Lightning web components (LWCs) and pages, your site preview automatically refreshes when source code changes are detected.
 
-- Basic HTML and CSS edits
-- Importing new CSS
-- Javascript edits in-service component library
-- Javascript method changes in the LWC component that don't alter its public API.
+When you edit local files in developer preview mode, your LWCs reflect these changes within your {org name} org:
 
-Other local changes may require deployment to your org. However, changes made directly in your org (like modifying component properties) are immediately live and won't show in your local files until you retrieve them from the org.
+- Basic HTML and CSS edits
+- Importing new CSS-only LWCs
+- JS edits in-service component library
+- JS method changes that don't affect the LWC's public API
 
-# flags.name.summary
+To see any other local changes in your site, you have to deploy to your org. If you make changes directly in your org, (like saving new component properties,) they're immediately live on your site. To update your local version of the site with those changes, you have to retrieve them from your org.
 
-Specify the site name for preview.
+Use the appropriate topic to preview specific aspects of the development environment.
 
-# flags.name.description
+# flags.name.summary
 
-The site name needs to match the name of a site on the current org. Example: "B2C CodeCept"
+Name of the Experience Cloud site to preview. It has to match a site name from the current org.
 
-# examples
+# flags.target-org.summary
```

Comment:
@nrkruk is this flag required for `sf lightning preview site`? I think it's required for `sf lightning preview app`.

Reply:
When I run the help for this command (i linked in your branch), I noticed that the summary for `--target-org` is different from that of `lightning preview app`.  This command still says: 

`  -o, --target-org=<value>  Username or alias of the target org.`

when it should say: 

`  -o, --target-org=<value>    (required) Username or alias of the target org. Not required if the `target-org` configuration variable is already set.`

The latter is the one that most core CLI commands use, and is better. Can you update this command so it shows the same summary? Thx.

Reply:
@jshackell-sfdc I have the latter/preferred text in the `app` and `site` files, but you're right that the CLI output shows the shorter text instead for `sf lightning preview site --help`. 
<img width="1208" alt="Screenshot 2024-07-02 at 10 50 47 AM" src="https://github.com/salesforcecli/plugin-lightning-dev/assets/93947656/7d52155d-c27e-4969-bcc2-fb89b9a8a559">

Not sure how to fix this -- could an engineer help us out?

Reply:
I _think_ the fix would be to update the definition of the `--target-org` flag in `site.ts` to this:

```
    'target-org': Flags.optionalOrg({
      summary: messages.getMessage('flags.target-org.summary'),
    }),
```

Currently it's just this:

`    'target-org': Flags.optionalOrg(),
`

Reply:
Just tried it, and yep, that'll do it!